.nvmrc

```diff
@@ -1 +0,0 @@
-0.10.40
```

Comment:
Correct, but if you're working in multiple node projects (like me), it's helpful to switch back and forth.

For instance, most of my other node projects, I dev at v4.1.x, but since Habit uses 0.10.x (for now), I added the .nvmrc so I could easily switch back and forth. If you already have that version installed, it's as easy as typing `nvm use`.